src/main/scala/io/iohk/ethereum/blockchain/sync/regular/BlockImporter.scala

```diff
@@ -329,7 +329,7 @@ class BlockImporter(
         Right(Nil)
       case UnknownBranch =>
         val currentBlock = blocks.head.number.min(startingBlockNumber)
-        val goingBackTo = currentBlock - syncConfig.branchResolutionRequestSize
+        val goingBackTo = (currentBlock - syncConfig.branchResolutionRequestSize).max(0)
```

Comment:
What is this change for? Reverting this doesn't seem to break anything. (Also goes for the `branchResolutionRequestSize = 30` changes.)

Reply:
before the change val goingBackTo could be set to negative numbers, this shouldn't be the case as the resolution of the branches can't start from before the genesis block.

I actually had this happening during tests -> hence the fix.

As for branchResolutionRequestSize values changes in tests, I just aligned them with the value we use in application.conf.